code/game/objects/items/storage/boxes/job_boxes.dm

```diff
@@ -59,10 +59,6 @@
 /obj/item/storage/box/survival/mining
 	mask_type = /obj/item/clothing/mask/gas/explorer/folded
 
-/obj/item/storage/box/survival/mining/PopulateContents()
-	..()
-	new /obj/item/crowbar/red(src)
-
```

Comment:
Pleaseeee don't remove this, miners may deal with ruins that need doors pried open, may need to pry their way into a powerless outpost that has no staff to maintain it, basically need to open doors on a powerless planet. It's not solely for modkits.

Reply:
why are you removing the crowbar

Reply:
yeah miners have more uses for crowbars than just their PKA